azerothcore/azerothcore-wotlk
# fix(DB/SAI): Add a few item restoration gossips/conditions/smart_scripts for Betina Bigglezink.

## Changes Proposed:
- Adds gossip menus, conditions, and smart_scripts for restoring [Dawn's Gambit](https://www.wowhead.com/wotlk/item=12368), [Seal of the Dawn](https://www.wowhead.com/wotlk/item=13209), and [Rune of the Dawn](https://www.wowhead.com/wotlk/item=19812) to [Betina Bigglezink](https://www.wowhead.com/wotlk/npc=11035) with the conditions of finishing the quests [Dawn's Gambit](https://www.wowhead.com/wotlk/quest=4771) and [The Active Agent](https://www.wowhead.com/wotlk/quest=5213).

## Issues Addressed:

- Closes #14466
- Closes #13157
- Closes https://github.com/chromiecraft/chromiecraft/issues/3746

## SOURCE:

Wowhead comments.

## Tests Performed:

- Tested in-game.

## How to Test the Changes:

1. `.quest add 4771`, `.quest add 5213`, `.quest reward 4771`, `.quest reward 5213`
2. Remove any vestigial items in your inventory.
3. Test the gossips and make sure they show up when they should.


## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.
